### ISSUE TITLE
Improve accessibility of search results

### ISSUE THREAD

**Issue by terrill** [ableplayer/ableplayer#347]
Search results are currently written to an unordered list, with the start time being the clickable element that triggers playback at the designated time in the video. The clickable element is currently tagged as a `<span>` with no ARIA, and fails to communicate to assistive technologies that it's clickable. Recommended improvements needed: 

- Tag the results as a `<button>`
- Add an aria-label that provides more context for the button text  (e.g., "Play at 1:10"). 
- Add an aria-describedby attribute to the button that references the text of the search results

**Comment by terrill** [comment]
This has been implemented in 6498838d43a0bcabe1be557b0f7ef3ed3cddb501 and 85ce66448891265817b15c5666af0c8b9c74dd23. 

In testing this with JAWS, NVDA, and VoiceOver, I discovered a bug in NVDA/Chrome in which NVDA fails to announce aria-describedby text that's contained with `<strong>` and `<em>` elements, which is true of the Able Player [video search demo](demos/video9.html). 

I created a [test page](http://terrillthompson.com/tests/aria/aria-describedby-inline.html) and filed [NVDA bug 10813](https://github.com/nvaccess/nvda/issues/10813).
